tasks/sep/mode-002-fp-upgrade/.env

```diff
@@ -0,0 +1,30 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+
+OWNER_SAFE=0x1Eb2fFc903729a0F03966B917003800b145F56E2
+COUNCIL_SAFE=0xf64bc17485f0B4Ea5F06A96514182FC4cB561977
+FOUNDATION_SAFE=0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B
+LIVENESS_GUARD=0xc26977310bC89DAee5823C2e2a73195E85382cC7
+
+# No SAFE_NONCE_* values are specified, we just use the current nonces
+# SAFE_NONCE_0X1EB2FFC903729A0F03966B917003800B145F56E2=
+# SAFE_NONCE_0XF64BC17485F0B4EA5F06A96514182FC4CB561977=
+# SAFE_NONCE_0XDEE57160AAFCF04C34C887B5962D0A69676D3C8B=
+
+# Expected prestate is cannon32 1.6.0-rc.2
+EXPECTED_PRESTATE=0x03513e996556589f633fe1d38d694f63bc93cca5df559af37631b30875a829e9
+
+# Expected Guardian address is the STANDARD config role Sepolia Guardian
+EXPECTED_GUARDIAN=0x7a50f00e8D05b95F98fE38d8BeE366a7324dCf7E
+
+# Contracts deployed prior to the upgrade
+EXPECTED_ANCHOR_STATE_REGISTRY_PROXY=0x5C30E13389686AE0ddF41859FbD4BD2CBE2Aaf1D
+EXPECTED_ANCHOR_STATE_REGISTRY_IMPL=0x43c0EA5cf1614a3F16b51804F3283f95d09a903f
+EXPECTED_PERMISSIONED_DISPUTE_GAME_IMPL=0xdC0c2Cb09512490B0b81292b9153415C3f092D4b
+EXPECTED_PERMISSIONED_DELAYED_WETH_PROXY=0xD92Bf98a4049097C4b2288734748dce90803E1C1
+EXPECTED_DISPUTE_GAME_FACTORY_PROXY=0x7Bb634B42373A87712Da14064deD13Db8b8b14f4
```

Comment:
This is an incorrect version of DisputeGameFactory.  It's version 1.0.0 not version 1.3.1.